libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/internal/extensions/MapboxCarMapSurfaceExtensions.kt

```diff
@@ -0,0 +1,33 @@
+@file:JvmName("MapboxCarMapSurfaceEx")
+@file:OptIn(MapboxExperimental::class)
+
+package com.mapbox.androidauto.car.internal.extensions
+
+import com.mapbox.maps.MapboxExperimental
+import com.mapbox.maps.Style
+import com.mapbox.maps.extension.androidauto.MapboxCarMapSurface
+import com.mapbox.maps.plugin.delegates.listeners.OnStyleLoadedListener
+
+internal fun MapboxCarMapSurface.handleStyleOnAttached(
```

Comment:
This is quite nice. Nice work! 

Thinking about this more, and I think we can move it up to the maps-androidauto extension
https://github.com/mapbox/mapbox-maps-android/tree/main/extension-androidauto

Happy to merge this here, and then we can propose it to maps. And eventually delete it from here